helmfile/charts/grafana-ops/dashboards/gatekeeper-dashboard.json

```diff
@@ -20,7 +20,7 @@
   "links": [],
   "panels": [
     {
-      "content": "\n# Detailed gatekeeper logs can be found in kibana:\n\n\n<<kibanaURL>>",
+      "content": "\n# Detailed gatekeeper logs can be found in OpenSearch Dashboards:\n\n\n<<kibanaURL>>",
```

Comment:
\n\n\n<<kibanaURL>>
Does kibanaURL matter here?

Reply:
Fixed it :slightly_smiling_face: